src/workflow/status.c

```diff
@@ -80,19 +81,9 @@ workflow_t* workflow_status(
 }
 
 /**
- * Callback used to wrap workflow_status into workflow_status_blocking().
- */
-static void _unlock_cb(void* param)
-{
-    (void)param;
-    workflow_blocking_unblock();
-}
-
-/**
- * Blocking wrapper for workflow_status().
+ * Blocking status.
  */
 void workflow_status_blocking(const char* msg, bool status_success)
 {
-    workflow_stack_start_workflow(workflow_status(msg, status_success, _unlock_cb, NULL));
```

Comment:
I'll do a bunch more such commits, and in the end we can remove all workflow_* things.